doc/maintaining/configuration.rst

```diff
@@ -1556,6 +1556,21 @@ Default value: (none)
 
 By default, the locales are searched for in the ``ckan/i18n`` directory. Use this option if you want to use another folder.
 
+.. _ckan.timezone:
+
+ckan.timezone
```

Comment:
@wardi well, [naming is considered to be hard](http://martinfowler.com/bliki/TwoHardThings.html) :smiley:  Thanks for this suggestion as well, I changed my code accordingly.